main.go

```diff
@@ -3,54 +3,33 @@ package main
 import (
 	"errors"
 	"flag"
-	"fmt"
-	"io/ioutil"
 	"log"
 	"os"
-
-	"gopkg.in/yaml.v2"
 )
 
-type argSlice []string
-
-func (c *argSlice) String() string {
-	return fmt.Sprintf("%s", *c)
-}
-
-func (a *argSlice) Set(value string) error {
-	*a = append(*a, value)
-	return nil
-}
-
-type config struct {
-	DepWarnning  string
-	Script       []string `yaml:"script"`
-	Build        []string `yaml:"build"`
-	IgnoredItems []string `yaml:"ignore"`
-	Verbose      bool     `yaml:"verbose"`
-}
-
 const (
 	Version       = "1.1.1"
 	VersionOutput = "Snag version " + Version
 )
 
-var (
-	cliCmds argSlice
-	version bool
-	verbose bool
-)
+const SnagFile = ".snag.yml"
 
 func init() {
-	flag.Var(&cliCmds, "c", "List of commands to execute")
-	flag.BoolVar(&verbose, "v", false, "Verbose output")
-	flag.BoolVar(&version, "version", false, "display snag's version")
+	log.SetOutput(os.Stdout)
+	log.SetFlags(0)
 }
 
 func main() {
 	flag.Parse()
+	if flag.NArg() > 0 {
+		if err := handleSubCommand(flag.Arg(0)); err != nil {
+			log.Fatal(err)
+		}
+		return
+	}
+
 	if version {
```

Comment:
I think we could actually amend it as part of this PR and deprecate the version flag. What do you think about having a separate package for all the different commands. I didn't want to make a new package for it since `init` was only one command. What are your thoughts?